prime-router/metadata/schemas/PrimeDataInput/pdi-covid-19.schema

```diff
@@ -22,184 +23,189 @@ elements:
 
-  - name: standard.test_performed_code
+  - name: test_performed_code
     csvFields: [{name: Ordered_test_code}]
 
-  - name: standard.specimen_source_site_code
+  - name: specimen_source_site_code
     csvFields: [{name: Specimen_source_site_code, format: $code}]
 
-  - name: standard.specimen_type
+  - name: specimen_type
     csvFields: [{name: Specimen_type_code}]
 
   # The type of device used for the test.  eg, BinaxNOW COVID-19 Ag Card
-  - name: standard.equipment_model_name
+  - name: equipment_model_name
     csvFields: [{name: Device_ID}]
 
   # Identifier for the specific piece of equipment used.
-  - name: standard.equipment_instance_id
+  - name: equipment_instance_id
     csvFields: [{name: Instrument_ID}]
 
-  - name: standard.testing_lab_specimen_id
+  - name: testing_lab_specimen_id
     csvFields: [{name: Testing_lab_specimen_ID}]
 
 # Removed until SimpleReport sends this column
-#  - name: standard.test_result_status
+#  - name: test_result_status
 #    csvFields: [{name: Test_result_status}]
 
-  - name: standard.test_result
+  - name: test_result
     csvFields: [{name: Test_result_code}]
 
-  - name: standard.illness_onset_date
+  - name: illness_onset_date
     csvFields: [{name: Illness_onset_date}]
 
-  - name: standard.specimen_collection_date_time
+  - name: specimen_collection_date_time
     csvFields: [{name: Specimen_collection_date_time}]
 
-  - name: standard.order_test_date
+  - name: order_test_date
     csvFields: [{name: Order_test_date}]
 
-  - name: standard.test_result_date
+  - name: test_result_date
     csvFields: [{name: Test_date, format: yyyyMMdd}]
 
-  - name: standard.date_result_released
+  - name: date_result_released
     csvFields: [{name: Date_result_released, format: yyyyMMdd}]
 
-  - name: standard.patient_race
+  - name: patient_race
     csvFields: [{name: Patient_race}]
 
-  - name: standard.patient_dob
+  - name: patient_dob
     csvFields: [{name: Patient_DOB}]
 
-  - name: standard.patient_gender
+  - name: patient_gender
     csvFields: [{name: Patient_gender}]
 
-  - name: standard.patient_ethnicity
+  - name: patient_ethnicity
     csvFields: [{name: Patient_ethnicity}]
 
-  - name: standard.patient_street
+  - name: patient_street
     csvFields: [{name: Patient_street}]
 
-  - name: standard.patient_street2
+  - name: patient_street2
     csvFields: [{name: Patient_street_2}]
 
-  - name: standard.patient_city
+  - name: patient_city
     csvFields: [{name: Patient_city}]
 
-  - name: standard.patient_state
+  - name: patient_state
     csvFields: [{name: Patient_state}]
 
-  - name: standard.patient_zip_code
+  - name: patient_zip_code
     csvFields: [{name: Patient_zip_code}]
 
-  - name: standard.patient_phone_number
+  - name: patient_phone_number
     csvFields: [{name: Patient_phone_number}]
 
-  - name: standard.patient_county
+  - name: patient_county
     csvFields: [{name: Patient_county}]
 
     # Eg, Staff, Resident, Visitor, Student
```

Comment:
Hmm.   Very similar to some of the AOE questions, eg, "Works in healthcare?" and "Lives in congregate setting?"....